setup.py

```diff
@@ -20,7 +20,6 @@ def find_stub_files(name: str) -> List[str]:
     readme = f.read()
 
 dependencies = [
-    "mypy>=0.991",
     "django-stubs>=4.2.5",
```

Comment:
We also have to bump `django-stubs` here.

Maybe we should use `django-stubs[compatible-mypy]` here? 
This way we can be sure that the version resolution works correctly for both packages.

Reply:
> We also have to bump django-stubs here.

Yep, I bump this dependency every time I make a release, e.g. https://github.com/typeddjango/djangorestframework-stubs/pull/494/files

> Maybe we should use django-stubs[compatible-mypy] here?

Nope, that would re-introduce the mypy dependency.

Reply:
Hmm, we *could* add "django-stubs[compatible-mypy]" to the extras deps, e.g.

```python
extras_require = {
    "compatible-mypy": ["mypy~=1.6.0", "django-stubs[compatible-mypy]"],
```

Reply:
yes, this is what I meant :+1:

Reply:
It has happened a few times that `django-stubs` and `djangorestframework-stubs` versions for `compatible-mypy` get out of sync.

My understanding is that pip's constraint resolver is pretty primitive. Not totally sure, but I'm afraid pip would just give up in such a situation, and make `djangorestframework-stubs[compatible-mypy]` uninstallable.

Testing this hypothesis is difficult though. So maybe err on the side of caution and not introduce this?

However, what does make sense is adding `django-stubs[compatible-mypy]` dependency to `requirements.txt`, so our CI will alert such an issue.

Reply:
I played around with this and seems like my fears were unfounded. When changing the dependencies around,  `pip` was successfully capable of finding the newest `django-stubs` with a compatble mypy dependency.

Let's go with that.